### PR TITLE
Summon new features 24.06.00

### PR DESCRIPTION
**Pull request by AlexanderBlanchardAC**

### CODE REVIEW

File: code/web/interface/themes/responsive/images/summon.png
Comment (AlexanderBlanchardAC) [inline code/web/interface/themes/responsive/images/summon.png:1]: This image is used for the Summon options of the Explore More Bar.

File: code/web/interface/themes/responsive/Search/explore-more-bar.tpl
Comment (AlexanderBlanchardAC) [inline code/web/interface/themes/responsive/Search/explore-more-bar.tpl:1]: The partner asked for the ability to remove the 'explore more' bar from the results pages. I could customise it more by giving the option to display it or hide it for different types of searches? e.g. Summon, library catalogue etc?

**Comment by AlexanderBlanchardAC** [comment]
I removed the zone:identifier file. Windows generated it with Windows specific metadata, but it seems to work without it.

File: code/web/interface/themes/responsive/Search/explore-more-bar.tpl
Comment (mdnoble73) [reply code/web/interface/themes/responsive/Search/explore-more-bar.tpl:1]: Yeah I think we'd want to do it based on the type of search and also be able to control what options show within Explore more as well as the order of the options showing in explore more.  Do we want to split that out into another PR?

**Pull request closed without merging**